messenger.rst

```diff
@@ -19,6 +19,27 @@ install messenger before using it:
 
     $ composer require messenger
 
+Message
+-------
+
+Before you can send a message, you must create it first. There is no specific
+requirement for a message::
```

Comment:
Actually, there are some requirements for a message to be serializable by the Serializer component (see https://github.com/symfony/symfony/issues/29163)
But that's going to be hard to explain in details.
Maybe just tell that a message should be serializable and unserializable by a Symfony Serializer instance - which should be the case for most plain PHP objects?